src/core_plugins/metrics/server/lib/get_vis_data.js

```diff
@@ -0,0 +1,60 @@
+import _ from 'lodash';
+import moment from 'moment';
+import Color from 'color';
+import calculateLabel from '../../public/components/vis_editor/lib/calculate_label';
```

Comment:
Cool... I'm moving a bunch of stuff around today that will take care of most of the file layout issues you mentioned.